src/chrome/content/rules/Moverall.com.xml

```diff
@@ -0,0 +1,15 @@
+<ruleset name="Moverall">
+
+	<target host="moverall.com" />
+	<target host="*.moverall.com" />
+
+	<test url="http://moverall.com/" />
+	<test url="http://www.moverall.com/" />
+	<test url="http://cdn.moverall.com/" />
+	<test url="http://az698912.vo.msecnd.net/" />
```

Comment:
@jsha I thought that this would only apply this rule when the current url is matching the target hosts? As in, when a page patching the target url's then apply the following rules?

I'll add the host to the target's.